invenio_i18n/templates/invenio_i18n/macros/language_selector.html

```diff
@@ -0,0 +1,23 @@
+
+{% macro language_render() %}
+
+  {% for language_item in config['I18N_LANGUAGES'] %}
+    {{ caller("javascript:document.getElementById('form_invenio_i18n_lang_{0}').submit();".format(language_item[0]), language_item[1]) }}
+  {% endfor %}
+
+  {% for language_item in config['I18N_LANGUAGES'] %}
+    <form id="form_invenio_i18n_lang_{{ language_item[0] }}"
```

Comment:
There is even nicer HTML5 way:
```html
<form action="{{ url_for('invenio_i18n.set_lang')}}" method="POST">
  {% trans %}Choose your favorite language:{% endtrans %}
  <button name="lang_code" type="submit" value="cs">Čeština</button>
  <button name="lang_code" type="submit" value="en">English</button>
</form>
```
http://www.w3schools.com/tags/att_button_value.asp